htdocs/fourn/class/fournisseur.commande.class.php

```diff
@@ -470,7 +470,7 @@ public function fetch_lines($only_product = 0)
     	if ($only_product) $sql .= ' AND p.fk_product_type = 0';
 		if (!empty($conf->global->PRODUCT_USE_SUPPLIER_PACKAGING)) {
 			$sql.= " AND l.qty >= pfp.quantity ";
-			$sql.= " GROUP BY l.rowid HAVING max_qty = MAX(pfp.quantity) ";
+			$sql.= " GROUP BY pfp.rowid, l.rowid HAVING max_qty = MAX(pfp.quantity) ";
```

Comment:
I think this fix is not enough : If there is no supplier price defined on a product, the fetch_lines will always return no lines;
I have pushed a fix here  444abbbc0e85cc5b318f57578f0715aa656a07e1
but i think fix should be a merge of yours and mine. Can you try it to merge it ?